specification/keyvault/data-plane/TypeSpec/main.tsp

```diff
@@ -0,0 +1,34 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "./models.tsp";
+import "./routes.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.ResourceManager.Foundations;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.Versioning;
+@armProviderNamespace
+@service({
+    title: "KeyVaultClient"
+  })
+@versioned(Versions)
+/**
+* The key vault client performs cryptographic key operations and vault operations against the Key Vault service.
+*/
+namespace KeyVaultClient;
```

Comment:
We should be configuring the server endpoint as well: https://typespec.io/docs/libraries/http/reference/decorators#@TypeSpec.Http.server